arghadipmanna101/Flipkart_Clone
# More Sections added

# Pull Request: Addition of Top Deals and Discount Deals Sections

## Description
This PR adds more content to the Flipkart clone project, specifically adding sections for Top Deals and Discount Deals. The Top Deals section showcases featured products with significant discounts, while the Discount Deals section highlights ongoing promotions and discounts across various product categories.

## Changes Made
- Added HTML and CSS for the Top Deals section
- Added HTML and CSS for the Discount Deals section
- Updated the main HTML file to include the new sections
- Added corresponding images for the new deals sections
- Updated styling to ensure consistency with the existing design

## Screenshots:
![Screenshot (70)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/87803348/1b5eca8b-8dc8-4f24-9e13-b7bb4a79919c)
![Screenshot (71)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/87803348/06ee2caf-7c16-4fd9-bff1-4521fd062bc0)
![Screenshot (72)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/87803348/5f1a042b-c689-4900-ac44-1ce10c268485)
![Screenshot (73)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/87803348/d302c713-727b-4daf-9b7e-b36676b857d0)


## Checklist
- [x] Tested locally to ensure the new sections display correctly
- [x] Ensured the code follows best practices and is properly commented
- [x] Updated documentation and comments to reflect the changes made
- [x] Ensured compatibility with different screen sizes and browsers

## Additional Notes
This PR is under GSSOC'24

## Resolving Issue
#295 

## How to Test
1. Checkout this branch locally.
2. Run the application and navigate to the corresponding sections to view the new Top Deals and Discount Deals.
3. Verify that the layout is responsive and displays correctly on different devices.
4. Check for any rendering issues or inconsistencies in styling.
